lib/std/http/status.zig

```diff
@@ -0,0 +1,203 @@
+//! HTTP Status
+//! https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
+
+const std = @import("../std.zig");
+
+pub const Status = union(enum) {
+    standard: Standard,
+    unknown: Unknown,
+
+    pub const Standard = enum(u16) {
```

Comment:
Why `u16`?
For HTTP 1.x syntax compat, status can only be 100-999; i.e. 899 possible values, which would fit in a `u10`

Reply:
`u10` is an odd bit size, didn't know if it would hurt perf